Packs/MapValuesTransformer/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "MapValuesTransformer",
+    "description": "Provides a transformer script to map a value that is present in one list, to a value in another list.",
+    "support": "xsoar",
```

Comment:
I think it is better to move this transformer to CommonScripts pack. A lot of users will benefit from it